src/components/help-guide.js

```diff
@@ -256,6 +256,15 @@ export class HelpGuide {
    * @return {HTMLTemplate}
    */
   createTemplate(state) {
+    // ----- This code is used to scroll to "Food with extra requirements"
+    // Section in Help guide ----- //
+    const helpGuideButton = document.querySelector("button[data-accordion-item='help-guide']");
+    if (helpGuideButton) {
+      helpGuideButton.addEventListener('click', () => {
+        document.getElementById('foodWithExtraReq').scrollIntoView();
```

Comment:
swap from ById to document.getElementsByClassName

Reply:
Changed to getElementsByClassName().